src/TicTacToe.java

```diff
@@ -0,0 +1,87 @@
+public class TicTacToe {
+
+    private Piece[][] board = new Piece[3][3];
+    private boolean gameIsFinished = false;
+
+    public TicTacToe() {
+    }
+
+
+
+    public boolean isAllEqual(Piece[] array){
+        for (int i = 0; i < array.length; i++){
+            if(array[0].getPieceType() != array[i].getPieceType()){
+                return false;
+            }
+        }
+        return true;
+    }

```

Comment:
```suggestion
    public boolean isAllEqual(Piece[] array){
        char pieceType = array[0].getPieceType();
        for (int i = 0; i < array.length; i++){
            if(pieceType != array[i].getPieceType()){
                return false;
            }
        }
        return true;
    }

```
Je hoeft `array[0].getPieceType()` maar 1x te doen, hoeft niet elke keer in de for-loop

Reply:
Absolutely excellent point!
Cheers!